### DISCUSSION TITLE
When does the DB get updated with new assets?

### DISCUSSION THREAD

**Discussion by alexpiet** [AllenNeuralDynamics/aind-data-access-api#81] [General]
My use case if checking if assets were correctly uploaded and processed overnight. This morning it appears that the assets were created between 11pm and midnight, yet this morning the DB doesn't contain any information about the.

When does the DB get updated, and how frequently?

**Comment by dyf** [comment]
@alexpiet currently every 12 hours (@jtyoung84 please confirm).  We can change this cadence.

**Reply by alexpiet** [reply]
That cadence is fine, if you tell me when so I can run my script after

**Reply by alexpiet** [reply]
Note from in-person discussion: Possibly increase the cadence, and make sure one of the updates happens sometime in the morning before the start of the work day (7-9 am?)

**Reply by dyf** [reply]
Thanks @alexpiet, I'll create an issue.

**Reply by jtyoung84** [reply]
The cadence is set in a different repo. I added a feature request there: https://github.com/AllenNeuralDynamics/aind-aws-infrastructure/issues/226
It's scheduled to run at 3am and 3pm, but I'd have to double check if that's UTC or local.

**Reply by helen-m-lin** [reply]
It currently runs 3am and 3pm UTC so 8am/pm pst

**Reply by helen-m-lin** [reply]
@alexpiet just FYI that the cadence was increased last monday. The indexer now runs every 4h.
